packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx

```diff
@@ -148,17 +160,17 @@ class ColorPalette extends Component<ColorPaletteProps, ColorPaletteState> {
   applyBoundaries(x: number, y: number) {
     let newXPosition = x
     let newYPosition = y
-    if (x > this.paletteWidth) {
-      newXPosition = this.paletteWidth
+    if (x > this.paletteWidth - this.padding) {
```

Comment:
Because now we have the padding so instead of moving from 0 to `props.width` or `props.height`, the indicator now will move from `padding` to `props.width-padding`